v3/plugins/redhat/vscode-openshift-connector/0.0.21/meta.yaml

```diff
@@ -14,5 +14,6 @@ spec:
   containers:
     - image: "eclipse/che-remote-plugin-openshift-connector-0.0.21:next"
       name: "vscode-openshift-connector"
+      memoryLimit: "512Mi"
```

Comment:
I'm not sure what is our strategy of supporting previous versions of plugins, but in this particular case looks like it cost nothing if we set memory limit for others `0.0.17` and `0.0.19` versions but it may prevent OOM if somebody uses not the latest plugin in his Devfile.

Reply:
makes sense
the PR has been updated

Reply:
:+1: